### PR TITLE
second pass at dashboard formatters

### LINKED ISSUE #194: Using formatters in dashboards
# Bug Report

## Description
The `[formatters]` option works for a stand-alone `<google-chart>` but not within a `<dashboard>`.

(Please don't take these issues as complaints - I really like angular-google-charts.)

## To reproduce
1. Create charts with formatters within and outside a dashboard:
2. Check which charts run formatters - dashboard chart doesn't, standalone chart does.

<pre><code>
  &lt;dashboard  [data]="data" [columns]="[ 'Team', 'Question', 'Start Time', 'End Time' ]" class="timeline" &gt;
      &lt;control-wrapper  [for]="[tChart]" type="DateRangeFilter"&gt;&lt;/control-wrapper&gt;
      &lt;google-chart     #tChart type="Timeline" [dynamicResize]="true"
                        [options]="opts"
                        [formatters]="formatters">&lt;/google-chart&gt; &lt;!-- NOT USED --&gt;
  &lt;/dashboard&gt;
  &lt;google-chart     type="Timeline" [dynamicResize]="true"
                    [data]="data" [columns]="[ 'Team', 'Question', 'Start Time', 'End Time' ]"
                    [options]="opts"
                    [formatters]="formatters" >&lt;/google-chart&gt; &lt;!-- USED --&gt;
</code></pre>

## Expected behaviour
The `[formatters]` should work the same inside or outside a dashboard.

## Your environment
<pre><code>
Angular CLI: 11.0.4
Node: 10.16.3
OS: linux x64

Angular: 11.0.4
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1100.4
@angular-devkit/build-angular   0.1100.4
@angular-devkit/core            11.0.4
@angular-devkit/schematics      11.0.4
@angular/cdk                    11.0.2
@angular/material               11.0.2
@schematics/angular             11.0.4
@schematics/update              0.1100.4
rxjs                            6.6.3
typescript                      4.0.5
</code></pre>

### PR DESCRIPTION
**Pull request by kussmaul**
copied bits of formatter code from google-chart.comp.ts to dashboard.comp.ts
added formatter to test.comp.ts and test.comp.html to test/demo
fixes #194

### CODE REVIEW

**Comment by kussmaul** [comment]
Dashboard formatters are even easier with data-table.service. I also included 2 unit tests.

**Comment by kussmaul** [comment]
addresses #194 : Using formatters in dashboards

File: apps/playground/src/app/test/test.component.ts [hunk 1]
```diff
@@ -74,4 +74,14 @@ export class TestComponent {
   public goBack() {
     this.location.back();
   }
+
+  public dashboardFormatters() {
+    // const cf          = new google.visualization.ColorFormat ();
```
Comment (FERNman) [inline apps/playground/src/app/test/test.component.ts:79]: Please remove the comments, I guess they are not needed.

**Review by FERNman — CHANGES_REQUESTED** [review]
Looks very good already! In addition to the code change, could you add a paragraph to the readme explaining the use of formatters in dashboards? After that, I'm happy to merge it 🙂

File: apps/playground/src/app/test/test.component.ts (hunk 1 above)
Comment (kussmaul) [reply apps/playground/src/app/test/test.component.ts]: Done, and done.

**Review by FERNman — APPROVED** [review]

**Pull request merged**